packages/common/src/content/search.ts

```diff
@@ -45,22 +47,10 @@ export async function enrichContentSearchResult(
       siteRelative: "not-implemented",
       thumbnail: "not-implemented",
     },
-    location: item.properties?.location,
+    location: deriveLocationFromItem(item),
```

Comment:
The only problem with this is that Projects and Sites do not go through this function... We may want to consider abstracting this higher up or duplicating this line across `enrichSiteSearchResult` and `enrichProjectSearchResult`.